exams/management/commands/authorization_user_exam.py

```diff
@@ -32,41 +28,29 @@ def add_arguments(self, parser):
             type=int
         )
         parser.add_argument(
-            "--course-key",
-            dest="course_key",
-            default=None,
-            help="Edx course key where we want to authorize users who passed course(s).",
-            type=int
+            "--all",
+            dest="all",
+            default='n',
+            help="Authorize eligible users for exams. Can be y or n",
+            type=str
         )
 
     def handle(self, *args, **options):
         """
         Trigger exam authorizations for user for a course if he has passed and paid for it.
         """
-        # pylint: disable=too-many-locals
+        all = options.get("all", 'n')
         program_id = options.get("program_id", None)
         username = options.get("username", None)
```

Comment:
question: are you sure you need to specify the default if there is a `default=` option when you add the arguments before?